apisix/plugins/dynamic-upstream.lua

```diff
@@ -305,11 +305,17 @@ function _M.access(conf, ctx)
         core.log.info("conf.rules: ", core.json.encode(conf.rules))
         for _, single_match in pairs(rule.match) do
             for _, var in pairs(single_match.vars) do
-                local op = var[2]
+                -- match is empty
+                local req_v, op, def_v = var[1], var[2], var[3]
```

Comment:
add a comment:

```
["http_host", "==", "localhost"]
["http_host", "in", ["localhost"]]
```

Reply:
`left_v`, `right_v` is a better name here

Reply:
Used another way, the variables here have been removed(req_v、def_v ).